.github/workflows/docker.yml

```diff
@@ -1,32 +1,38 @@
 name: Publish Docker image
 on:
-  release:
-    types: [published]
+  # release:
+  #   types: [published]
+  push:
```

Comment:
add pull_request trigger like https://github.com/PrismarineJS/flying-squid/blob/master/.github/workflows/ci.yml#L6 to be able to test